kong.conf.default

```diff
@@ -178,6 +178,9 @@
 # See http://nginx.org/en/docs/http/ngx_http_realip_module.html#real_ip_recursive
 # for a description of this directive.
 
+#response_format = text/plain # The default response format to use when the
+                              # 'Accept' header is missing in the request
```

Comment:
would be nice here to have a note of the valid elements based on the defined enum. Also, this needs clarification on exactly when this is used (e.g. for Nginx error handling, and not always errors generated by Kong, by plugins, or by upstream services).